includes/Settings/Api/AccountSettingsService.php

```diff
@@ -274,4 +301,42 @@ private function saveSettingsToDatabase(Collection $settings): bool
 
         return update_option(AccountSettings::WP_OPTION_KEY, $array);
     }
+
+    /**
+     * @param  \WP_REST_Request $request
+     *
+     * @return void
+     * @throws \Exception
+     */
+    public function updateOrderStatus(WP_REST_Request $request): void
+    {
+        $requestBody = $request->get_body();
+        $jsonBody    = json_decode($requestBody, true);
+        $orderId     = $jsonBody['data']['hooks'][0]['order'] ?? null;
```

Comment:
Is het mogelijk om in de variabelenaam duidelijk te maken dat dit de uuid is die de api eraan heeft gegeven, en dus niet 1 van de orderIds is binnen wooc zelf? en het klopt dus echt dat deze uuid in het veld 'order' zit? ['data']['hooks'][0]['order'], niet ['order']['identifier'] ofzo? Het lijkt onlogisch.